src/registerPage.js

```diff
@@ -35,57 +29,52 @@ class RegisterForm extends Component {
     }
   };
 
-  RegisterForm = (
+  // User Registration form component
+  const RegisterForm = (
     <div style={{ marginTop: 200, marginLeft: 600 }}>
-      <FormControl
-        onSubmit={this.handleSubmit}
-        sx={{ border: 1, borderRadius: 2 }}
-      >
-        <h1> Registeration Form</h1>
-        <TextField
-          required
-          id="outlined"
-          multiline
-          label="Email"
-          type="email"
-          name="email"
-          placeholder="Email"
-          style={{ margin: 10 }}
-        />
-        <TextField
-          required
-          id="outlined"
-          multiline
-          label="Password"
-          type="password"
-          name="password"
-          placeholder="Password"
-          style={{ margin: 10 }}
-        />
-        <br />
-        <Button
-          type="Submit"
-          variant="contained"
-          style={{ marginLeft: 40, marginRight: 40 }}
+      <form onSubmit={handleSubmit}>
+        <FormControl
+          sx={{ border: 1, borderRadius: 2 }}
         >
-          Register
-        </Button>
-      </FormControl>
+          <h6> Registeration Form</h6>
+          <TextField
+            required
+            id="outlined"
+            multiline
+            label="Email"
+            type="email"
+            name="email"
+            placeholder="Email"
+            style={{ margin: 10 }}
+          />
+          <TextField
+            required
+            id="outlined"
+            multiline
+            label="Password"
+            type="password"
+            name="password"
+            placeholder="Password"
+            style={{ margin: 10 }}
+          />
+          <br />
+          <Button
+            type="Submit"
+            variant="contained"
+            style={{ marginLeft: 40, marginRight: 40 }}
+          >
+            Register
+          </Button>
+        </FormControl>
+      </form>
     </div>
   );
 
-  render() {
-    console.log(this.props);
-    return (
-      <div className="App">
-        {this.state.isSubmitted ? (
-          <div>Registered Successfully</div>
-        ) : (
-          <>{this.RegisterForm}</>
-        )}
-        ;
-      </div>
-    );
-  }
+  return (
+    <div className="App">
+      {isSubmitted ? <div>Registered Successfully</div> : <>{RegisterForm}</>};
```

Comment:
instead when registered successfully show an alerty that registered succesfully then 
redirect backto login page